mozdownload/scraper.py

```diff
@@ -870,13 +870,130 @@ def platform_regex(self):
         return PLATFORM_FRAGMENTS[self.platform]
 
 
+class TryScraper(Scraper):
+    "Class to download a try build from the Mozilla server."
+
+    def __init__(self, changeset=None,
+                 debug_build=False, *args, **kwargs):
+
+        self.debug_build = debug_build
+        self.changeset = changeset
+
+        Scraper.__init__(self, *args, **kwargs)
+
+    def get_build_info(self):
+        "Defines additional build information"
+
+        self.build_index = None
+
+        self.locale_build = self.locale != 'en-US'
```

Comment:
I don't like expressions like this, as I feel python should be about readability (This is a personal preference).

On a separate note, except for b2g builds (where it is `multi`), the default locale is en-US anways. What is the reason for the last part of this line?